ca/ca.go

```diff
@@ -415,7 +414,7 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(ctx context.Context, csr x5
 	serialBytes[0] = byte(ca.prefix)
 	_, err = rand.Read(serialBytes[1:])
 	if err != nil {
-		err = core.InternalServerError(err.Error())
+		err = berrors.InternalServerError("boulder/ca.IssueCertificate: failed to generate serial: %s", err)
```

Comment:
No real argument for why I used the `boulder/` prefix, I'm fine removing it.

A method name variable may make sense, although I'd probably use a function local `var` in order not to pollute the global namespace any more.